lemmy_apub/src/activity_queue.rs

```diff
@@ -189,10 +189,6 @@ where
     return Ok(());
   }
 
-  for to_url in &to {
-    check_is_apub_id_valid(&to_url)?;
-  }
-
```

Comment:
Everything that calls this already does this check.

Reply:
I thought this would be good to have as kind of a second line of defense, in case we forget to check in the calling methods. But yes its not necessary.